articles/quickstart/spa/vuejs/interactive.md

```diff
@@ -0,0 +1,180 @@
+* Auth0 redirects you to your application using the value of the `redirectUri` you used to configure the plugin.
+
+:::
+
+:::checkpoint-failure
+Sorry about that. Here's a couple things to double check:
+* you configured the correct `redirectUri`
+* make sure the domain and Client ID are set correctly
+
+Still having issues? Check out our [documentation](https://auth0.com/docs) or visit our [community page](https://community.auth0.com) to get more help.
+
+:::
+::::
+
+## Add logout to your application {{{ data-action=code data-code="logout.js" }}}
+
+Users who log in to your project will also need a way to log out. When users log out, they will be redirected to your [Auth0 logout](https://auth0.com/docs/api/authentication?javascript#logout) endpoint, which will then immediately redirect them to specified `returnTo` parameter.
+
+Use the `logout` function that is exposed on the return value of `useAuth0`, which you can access in your component's `setup` function, to log the user out of your application.
+
+:::note
+To log the user out of your application but not from Auth0, use `logout({ localOnly: true })`.
+:::
+
+### Using the Options API
+If you're using the Options API, you can use the same `logout` method from the global `$auth0` property through the `this` accessor.
+
+::::checkpoint
+
+:::checkpoint-default
+
+Run your application and click the logout button, verify that:
+* your Vue application redirects you to the `returnTo` address
+* you are no longer logged in to your application
+
+:::
+
+:::checkpoint-failure
+Sorry about that. Here's a couple things to double check:
+* you specified a value for `returnTo` when calling `logout`
+* the `returnTo` value is configured as one of the Allowed Logout URLs in your Application Settings
+
+Still having issues? Check out our [documentation](https://auth0.com/docs) or visit our [community page](https://community.auth0.com) to get more help.
+
+:::
+
+::::
+
+## Show user profile information {{{ data-action=code data-code="profile.js" }}}
+
+Now that your users can log in and log out, you will likely want to be able to retrieve the [profile information](https://auth0.com/docs/users/concepts/overview-user-profile) associated with authenticated users. For example, you may want to be able to display a logged-in user’s name or profile picture in your project.
+
+Once the user authenticates, the SDK extracts the user's profile information and stores it in memory. It can be accessed by using the reactive `user` property exposed by the return value of `useAuth0`, which you can access in your component's `setup` function.
+
+Because the `user` property contains sensitive information related to the user's identity, its availability depends on the user's authentication status. To prevent render errors, you should always:
+- use the `isAuthenticated` property to determine whether Auth0 has authenticated the user before Vue renders any component that consumes the `user` property.
```

Comment:
```suggestion
The `user` property contains sensitive information related to the user's identity. It is only available based on the user's authentication status. To prevent render errors, you should always:
- use the `isAuthenticated` property to determine whether Auth0 has authenticated the user before Vue renders any component that consumes the `user` property.

```